utils/gen_onnx_mlir.py

```diff
@@ -297,6 +297,8 @@
     'SequenceInsert',
     'SpaceToDepth',
     'TopK',
+    "RoiAlign",
```

Comment:
We typically add them in alphabetical order... can be fixed later if you want to drop it now